flickit-assessment-data/src/main/java/org/flickit/assessment/data/jpa/kit/user/UserJpaRepository.java

```diff
@@ -9,4 +10,12 @@ public interface UserJpaRepository extends JpaRepository<UserJpaEntity, UUID> {
 
     Optional<UserJpaEntity> findByEmailIgnoreCase(String email);
 
+    @Query("""
+        SELECT u
+        FROM UserJpaEntity u
+        RIGHT JOIN ExpertGroupAccessJpaEntity ea on u.id = ea.userId
+        RIGHT JOIN ExpertGroupJpaEntity e on e.id = ea.expertGroupId
+        """)
+    Optional<UserJpaEntity> findUsersOfExpertGroups();
```

Comment:
This method has no usage and can be removed.

Reply:
Done